docs/compliance/cis/docker_ee.md

```diff
@@ -11,7 +11,7 @@ This includes Docker EE Engine, Docker Trusted Registry (DTR), and Universal
 Control Plane (UCP). These updates will also include configuration items for
 Docker EE Engine on Windows Server, Windows Server host items, and Windows-based
 Docker image recommendations. We anticipate completion of the draft and final
-update of the Benchmark in the February timeframe.
+update of the Benchmark at some point in the coming months.
```

Comment:
@anweiss  ... maybe you should rephrase rather than remove. Afterall, what does "coming months" mean? The context  here is January and its almost May ... So, maybe say in 2018? By year end? not sure, but something that the reader can rely on.

Reply:
Tweaked in follow-on commit